llvm/include/llvm/IR/RepoDefinition.h

```diff
@@ -169,17 +179,14 @@ class ModuleHashGenerator {
   template <typename GlobalType> void calculateGOInfo(const GlobalType *G);
   void calculateGOInfo(const GlobalObject *GO);
 
-  /// Accumulate the GO's digest and get its GODigestState.
+  /// Accumulate the GO's digest and get its Dependencies.
   /// \param GO Calculated global object.
   /// \param GOHash A object that is used to calculate the GO's hash value.
   /// \param UseRepoDefinitionMD true if using the digest of GO's dependencies
   ///        stored in the RepoDefinition metadata.
   /// \returns a structure containing the GO's state.
```

Comment:
I think the comment on the return value needs updating.

Reply:
Fixed, thanks.